### DISCUSSION TITLE
AI Deck bootloader problem

### DISCUSSION THREAD

**Discussion by coo122567** [bitcraze/discussions#444] [Q&A]
Hi everyone, 
what is the problem in the picture below? Use a virtual machine to do it.
![image](https://user-images.githubusercontent.com/118537645/204550047-117cd1f2-1d41-4400-b742-c00c6a053bde.png)

**Comment by noahzn** [comment]
Your device is not correctly connected. Please restart your crazyflie and then plug in your USB.

**Comment by knmcguire** [comment]
WAIT! before you do this, make sure to know if this is necessary. Make sure that you check the writings on the aideck board if you have an older version of the AIdeck (1.0 or 1.1 REV A-C). If you have an AIdeck 1.1 REV D, flashing the gap8 bootloader should not be necessary.
